pom.xml

```diff
@@ -111,7 +111,7 @@
         <repository>
           <id>sarxos</id>
           <name>sarxos</name>
-          <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+          <url>http://oss.sonatype.org/content/repositories/snapshots</url>
```

Comment:
Pls do not change back to http... This will break compatibility to newer mvn versions (maven-default-http-blocker)

Reply:
@holgerfriedrich I see what happened, MRL dynamically generates a POM file from a template, #1116 only changed pom.xml and not the configuration used to generate the repositories, so when it was regenerated the change was wiped out. I think a new PR should be opened to change to https, that would be in `org.myrobotlab.framework.repo.Repo`:

https://github.com/MyRobotLab/myrobotlab/blob/6e24be329e36209e5d4a9af89c56637faa504b12/src/main/java/org/myrobotlab/framework/repo/Repo.java#L135-L152

Reply:
@holgerfriedrich I updated the repo config and regenerated the pom, it should be using https again